protractor/helpers/reporter.ts

```diff
@@ -0,0 +1,8 @@
+import { SpecReporter, StacktraceOption } from 'jasmine-spec-reporter';
+export let reporter = () => {
```

Comment:
leave a blank line between import statements and the reporter definition.

Reply:
Done